tests/test_platform.py

```diff
@@ -99,67 +103,58 @@ def test_pypy_abi_prefix():
   )
 
 
-# NB: Having to patch here is a symptom of https://github.com/pantsbuild/pex/issues/694
-# Kill when the Platform API is fixed to not need to consult the local interpreter.
 @patch('pex.pep425tags.get_extension_suffixes', lambda: ['.abi3.so'])
 def test_platform_supported_tags_abi3():
-  tags = Platform.create('linux-x86_64-cp-37-m').supported_tags()
-  expected_tags = [
-    ('cp37', 'cp37m', 'linux_x86_64'),
-    ('cp37', 'cp37m', 'manylinux1_x86_64'),
-    ('cp37', 'abi3', 'linux_x86_64'),
-    ('cp37', 'abi3', 'manylinux1_x86_64'),
-    ('cp37', 'none', 'linux_x86_64'),
-    ('cp37', 'none', 'manylinux1_x86_64'),
-    ('cp36', 'abi3', 'linux_x86_64'),
-    ('cp36', 'abi3', 'manylinux1_x86_64'),
-    ('cp35', 'abi3', 'linux_x86_64'),
-    ('cp35', 'abi3', 'manylinux1_x86_64'),
-    ('cp34', 'abi3', 'linux_x86_64'),
-    ('cp34', 'abi3', 'manylinux1_x86_64'),
-    ('cp33', 'abi3', 'linux_x86_64'),
-    ('cp33', 'abi3', 'manylinux1_x86_64'),
-    ('cp32', 'abi3', 'linux_x86_64'),
-    ('cp32', 'abi3', 'manylinux1_x86_64'),
-    ('py3', 'none', 'linux_x86_64'),
-    ('py3', 'none', 'manylinux1_x86_64'),
-    ('cp37', 'none', 'any'),
-    ('cp3', 'none', 'any'),
-    ('py37', 'none', 'any'),
-    ('py3', 'none', 'any'),
-    ('py36', 'none', 'any'),
-    ('py35', 'none', 'any'),
-    ('py34', 'none', 'any'),
-    ('py33', 'none', 'any'),
-    ('py32', 'none', 'any'),
-    ('py31', 'none', 'any'),
-    ('py30', 'none', 'any'),
-  ]
-  assert expected_tags == tags
-
-
-# NB: Having to patch here is a symptom of https://github.com/pantsbuild/pex/issues/694
-# Kill when the Platform API is fixed to not need to consult the local interpreter.
+  assert_tags(
+    'linux-x86_64-cp-37-m',
+    [
+      ('cp37', 'cp37m', 'linux_x86_64'),
```

Comment:
Should `cp38` be added now that it's officially released?

Reply:
Perhaps, but then the platform would need to change. The algorithm / correct answer is a countdown from 3.minor to 3.2 for abi3 wheels. Counting up to some arbitrary future minor cweiling is not a thing.

I'll defer 3.8 support to another PR, there is more that needs to be done in CI and tox.